newdle/client/src/reducers/answer.js

```diff
@@ -0,0 +1,26 @@
+import {combineReducers} from 'redux';
+import {ADD_ANSWER, REMOVE_ANSWER, SET_ANSWER_ACTIVE_DATE} from '../actions';
+
+export default combineReducers({
+  answers: (state = {}, action) => {
+    switch (action.type) {
+      case ADD_ANSWER:
+        return {...state, [action.timeslot]: action.answer};
+      case REMOVE_ANSWER: {
+        const {[action.timeslot]: value, ...answers} = state;
```

Comment:
this looks broken, what's `value`?

Reply:
It works, I'm basically removing one option from answers this way. But probably there's a better way to do that

Reply:
Oh right, you are destructuring here and not returning the object.

Reply:
But what does "removing an answer" mean? Isn't this the same as being unavailable?

Reply:
Yep, I simply wasn't sure which way to go with unavailable answers in the redux state. 
Right now I'm storing only accepted and ifneedbe and omitting the unavailable. I could store the unavailable as well, but then I would always need to fill the state with 'unavailable' for each newdle time slot at the begining.

Reply:
Just filling anything missing with unavailable when saving the answers would work as well - but when you edit your existing response, then unavailable slots *will* be explicitly set to unavailable as well, so any display logic needs to be able to handle explicitly-unavailable slots.

Reply:
I didn't take editing into account, I see your point.